src/main/java/uk/gov/hmcts/divorce/document/DocumentIdProvider.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.divorce.document;
+
+import org.springframework.stereotype.Component;
+
+import java.util.UUID;
+
+@Component
+public class DocumentIdProvider {
```

Comment:
can be a util class ?

Reply:
I put the ID generation in an Autowireable component so that it can be mocked out in the tests, I don't have to bother about random UUIDs